shell/platform/windows/window.h

```diff
@@ -299,6 +303,33 @@ class Window : public KeyboardManager::WindowDelegate {
 
   // Timer identifier for DirectManipulation gesture polling.
   const static int kDirectManipulationTimer = 1;
+
+  // Implements IRawElementProviderFragmentRoot when UIA is enabled.
+  std::unique_ptr<ui::AXFragmentRootWin> ax_fragment_root_;
+
+  // Delegate for Fragment.
+  std::unique_ptr<WindowAXFragmentRootDelegate> ax_fragment_delegate_;
+
+  // Allow WindowAXFragmentRootDelegate to access protected method.
+  friend class WindowAXFragmentRootDelegate;
+};
+
+// A delegate class to the window.
+class WindowAXFragmentRootDelegate : public ui::AXFragmentRootDelegateWin {
```

Comment:
Could we make the Windows accessibility bridge implement `ui::AXFragmentRootDelegateWin` instead? That seems like a better location for this logic.

Reply:
I will test that. I think `UiaReturnRawElementProvider` performs some copying, so I would just need to check that non-idempotent setup would not be repeated if we do that.

Reply:
> ... but FlutterWindowsEngine stores `AccessibilityBridge` rather than `AccessibilityBridgeWindows`.

This was necessary until a recent refactoring (`AccessibilityBridgeWindows` is new, previously there was only the `AccessibilityBridge`). We can make the engine store a `AccessibilityBridgeWindows` and skip casting magic.